renderer/components/Activity/Row.js

```diff
@@ -1,15 +1,15 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { FormattedDate } from 'react-intl'
 import { Box } from 'rebass/styled-components'
 import { Bar, Heading } from 'components/UI'
+import { FormattedDateTime } from 'containers/UI'
 
 const Row = ({ style, item, RowComponent }) => (
   <div style={style}>
     {item.title ? (
       <Box mt={4} pl={4}>
         <Heading.H4 fontWeight="normal">
-          <FormattedDate day="2-digit" month="short" value={item.title} year="numeric" />
+          <FormattedDateTime format="date" month="short" value={item.title} />
```

Comment:
Maybe this is what's causing the prop warning? What is `title`? Is it always a string or Date?

Reply:
Yeah looks like in some cases its not a date, so have updated the value validation to be: PropTypes.any, will that be fine? Doing so the error does go away, shall we format the value before sending or any validation is fine?

Reply:
I guess really the component supports dates passed as `string` (which is what `item.title` is). Probably should update component prop type definition to `oneOfType([PropTypes.number, PropTypes.string, PropTypes.instanceOf(Date)])`

Reply:
Yep, adding that works fine and also fixes the validation issue. Updated the PR